caluma-demo/config/environment.js

```diff
@@ -23,7 +23,10 @@ module.exports = function(environment) {
     APP: {
       // Here you can pass flags/options to your application instance
       // when it is created
-    }
+    },
+
+    languages: ["de", "fr", "en"],
```

Comment:
`fr` seems not to be ready yet for the form builder, so let us remove this for the moment.

Reply:
Ah right. I've removed `fr`.